lib/async.js

```diff
@@ -32,22 +32,23 @@ module.exports = function resolve(x, options, callback) {
     var readFile = opts.readFile || fs.readFile;
 
     var extensions = opts.extensions || ['.js'];
-    var basedir = opts.basedir || path.dirname(caller());
+    var y = opts.basedir || path.dirname(caller());
```

Comment:
Can this use a better name than "y"? I'd previously renamed this to "basedir" for clarity.

Reply:
Hi @ljharb I just basically left @jmm's code intact, but I can certainly change ```y``` to ```basedir``` instead. 

However, note that ```y``` is really just a temporary variable and not intended to have any significant meaning, I'm guessing from @jmm's code, because at the end of the day, what you want is, ```parent``` and ```y``` just splits up the derivation of parent into two parts for clarity.

Anyway, let me know whether you still want ```y``` to be ```basedir``` and I will make the change.

Reply:
At the time they made their PR, the variable was called "y" :-) please do change it.